test/extensions/filters/http/composite/composite_filter_integration_test.cc

```diff
@@ -33,20 +33,24 @@ using envoy::extensions::common::matching::v3::ExtensionWithMatcherPerRoute;
 using envoy::extensions::filters::http::composite::v3::ExecuteFilterAction;
 using envoy::type::matcher::v3::HttpRequestHeaderMatchInput;
 using test::integration::filters::SetResponseCodeFilterConfig;
+using test::integration::filters::SetResponseCodeFilterConfigDual;
 using test::integration::filters::SetResponseCodePerRouteFilterConfig;
+using test::integration::filters::SetResponseCodePerRouteFilterConfigDual;
 using xds::type::matcher::v3::Matcher_OnMatch;
 
 struct CompositeFilterTestParams {
   Network::Address::IpVersion version;
   bool is_downstream;
+  bool is_dual_factory;
```

Comment:
Extend the test params to support filters derived from FactoryBase (downstream only )and DualFactoryBase(dual filter)